index.js

```diff
@@ -4,25 +4,27 @@ const Koa = require('koa');
 const serve = require('koa-static');
 const logger = require('koa-logger');
 const fs = require('fs');
-const path = require('path')
+const path = require('path');
 
 const yargs = require('yargs/yargs');
 const { hideBin } = require('yargs/helpers');
-const argv = yargs(hideBin(process.argv)).argv;
+
```

Comment:
Codacy has a fix for the issue: [Use object destructuring.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
const {argv} = yargs(hideBin(process.argv));
```